Sources/VersionCheck/VersionCheck.swift

```diff
@@ -6,10 +6,21 @@
 //
 
 import FirebaseRemoteConfig
-
+import Helpers
 
 public struct VersionCheck {
-    fileprivate var remoteConfig: RemoteConfig
+    var fetch: (_ latestVersionKeyFromFirebase: String,
```

Comment:
Same question as in `Maintenance.swift`.

Reply:
Same answer